### PR TITLE
Remove legacy build methods

### PR DESCRIPTION
**Pull request by pipex**
Removes dindctl and `tools/sync.js` in favor of balenaos-in-container
and SSH tunneling respectively.

Change-type: patch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,136 +106,74 @@ restarted, or let nodemon handle the changes.
...
-./dindctl run --image balena/amd64-supervisor:master
-```
-
-This will setup a Docker-in-Docker instance with an image that runs the supervisor image. You can replace `:master` for a specific tag (see the [tags in Dockerhub](https://hub.docker.com/r/balena/amd64-supervisor/tags/)) to run
-a supervisor from a branch or specific version. The script will pull the image if it is not already available in your
-local Docker instance.
-
-If you want to develop and test your changes, you can run:
+To run the supervisor in a balenaOS-in-container instance, first follow the installation instructions on the [balenaOS-in-container repostitory](https://github.com/balena-os/balenaos-in-container/). Make sure
+you have the image configured in [development mode](https://www.balena.io/docs/reference/OS/overview/#development-vs-production-mode). After the image starts you should be able to use the [sync](#Sync) method described above for runnign a livepush supervisor on the docker container.
```
Comment (cywang117) [inline README.md:110]:
```suggestion
you have the image configured in [development mode](https://www.balena.io/docs/reference/OS/overview/#development-vs-production-mode). After the image starts you should be able to use the [sync](#Sync) method described above for running a `livepush` Supervisor on the docker container.
```

File: README.md [hunk 2]
```diff
@@ -256,11 +194,11 @@ which ensures that the environment is exactly the same.
 You can run specific tests quickly with the `test:fast` script by matching with test suites (describe) or test cases (it) using a string or regexp:
 
 ```sh
-npm run test:fast -- --grep spawnJournalctl
+npm run test:fast -- --g spawnJournalctl
```
Comment (cywang117) [inline README.md:197]:
```suggestion
npm run test:fast -- -g spawnJournalctl
```

File: README.md [hunk 3]
```diff
@@ -256,11 +194,11 @@ which ensures that the environment is exactly the same.
...
 
-npm run test:fast -- --grep "detect a V2 delta"
+npm run test:fast -- --g "detect a V2 delta"
 
-npm run test:fast -- --grep (GET|POST|PUT|DELETE)
+npm run test:fast -- --g (GET|POST|PUT|DELETE)
```
Comment (cywang117) [inline README.md:201]:
```suggestion
npm run test:fast -- -g (GET|POST|PUT|DELETE)
```

**Comment by cywang117** [comment]
LGTM!

**Comment by pipex** [comment]
@balena-ci I self certify!

**Pull request merged**